src/Couchbase.Extensions.Caching/CouchbaseCache.cs

```diff
@@ -2,41 +2,49 @@
 using System.Threading.Tasks;
 using Couchbase.Core;
 using Microsoft.Extensions.Caching.Distributed;
-using Microsoft.Extensions.Internal;
 using Microsoft.Extensions.Options;
 
 namespace Couchbase.Extensions.Caching
 {
+    public interface ICouchbaseCache : IDistributedCache
+    {
+        IBucket Bucket { get; }
+
+        IOptions<CouchbaseCacheOptions> Options { get; }
```

Comment:
Would it be cleaner to simply return CouchbaseCacheOptions here?  The getter implementation could just call _options.Value.